### ISSUE TITLE
githubissues notifier - use single notifier for multiple repositories

### ISSUE THREAD

**Issue by LeeU1911** [GoogleCloudPlatform/cloud-build-notifiers#189]
Current configuration only allows a single repository
```
      githubRepo: myuser/myrepo
```

In a larger scale where teams work on multiple repositories, it is undesirable to deploy multiple Cloud Run notifier for each individual repository (from both cost and maintenance perspective)

With a small tweak to infer repository from Build's information (which it has), we can deploy the notifier once and use it for multiple repositories

**Comment by prabenzom** [comment]
Hi, I just reviewed that PR, hopefully that resolves this issue?

**Comment by VuKrampHub** [comment]
@prabenzom Yes, indeed. I created this issue to surface the problem in case and used to track the PR in case someone search for the same feature

**Comment by VuKrampHub** [comment]
@prabenzom I have another commit to automatically set the assignee to the GitHub issue based on commit info. The added complexity is Cloud Build context itself doesn't contain this information and I have to look up via an additional call to GH API.

Is that something you would want to introduce in the upstream version? If yes, I'm more than happy to open another PR for it.

**Comment by VuKrampHub** [comment]
Also want to ask: I saw someone threw the idea of having community-contributed notifier in the past, is that still on the radar?
